ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -78,6 +78,61 @@ objects:
 
     containers:
 
+    - name: lease
+      image: registry.svc.ci.openshift.org/ci/boskoscli:latest
+      terminationMessagePolicy: FallbackToLogsOnError
+      resources:
+        requests:
+          cpu: 10m
+          memory: 10Mi
+        limits:
+          memory: 200Mi
+      volumeMounts:
+      - name: shared-tmp
+        mountPath: /tmp/shared
+      - name: cluster-profile
+        mountPath: /tmp/cluster
+      - name: artifacts
+        mountPath: /tmp/artifacts
+      env:
+      - name: CLUSTER_TYPE
+        value: ${CLUSTER_TYPE}
+      - name: CLUSTER_NAME
+        value: ${NAMESPACE}-${JOB_NAME_HASH}
+      command:
+      - /bin/bash
+      - -c
+      - |
+        #!/bin/bash
+        set -euo pipefail
+
+        trap 'rc=$?; CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi; if test "${rc}" -ne 0; then touch /tmp/shared/exit; fi; exit "${rc}"' EXIT
+
+        if [[ "${CLUSTER_TYPE:-}" != "aws" ]]; then
+          echo "[WARNING] No resource leasing is defined for \"${CLUSTER_TYPE}\" clusters"
+          exit 0
```

Comment:
oh, you need to touch `/tmp/shared/leased` in this branch too.  Or adjust the `setup` container's block to only block on `leased` for `aws`.  Seems DRYer to touch `leased` here, even if we aren't actually leasing anything for those other platforms.

Reply:
fixed in https://github.com/openshift/release/pull/4553/files#diff-1dbbb3c9b7788155db302b2a277ab26aR122